apps/backend/src/db/migrations/20260731140000_message_conversation_state.sql

```
@@ -0,0 +1,28 @@
+  -- when a later pass (or a user) re-files it, so the row never points at a
+  -- conversation the message has left.
+  conversation_id TEXT NOT NULL,
+  state TEXT NOT NULL,
+  settled_by TEXT,
+  settled_at TIMESTAMPTZ,
+  created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+  updated_at TIMESTAMPTZ NOT NULL DEFAULT NOW()
+);
+
+CREATE INDEX idx_message_conversation_state_conversation
+  ON message_conversation_state (conversation_id, state);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)